.travis.yml

```diff
@@ -5,18 +5,24 @@ addons:
     packages:
       - graphviz
 
-python:
-  - "3.6"
-  - "3.7"
+matrix:
+  include:
+    - python: 3.6  
+      env: TOXENV=py36-test
+    - python: 3.7
+      env: TOXENV=py37-test
+    - python: 3.8
+      env: TOXENV=py38-test
+    - python: 3.6
```

Comment:
Do you really need three different doc builds?

Reply:
No just testing out :)